server/helpers/tripHelper.js

```diff
@@ -0,0 +1,161 @@
+import models from '../sequelize/models';
+
+const {
+  Trip,
+  User
+} = models;
+
+/**
+ * This class contains
+ * all methods required to save/edit/retrieve
+ * the trip data
+ */
+class TripHelpers {
+  /**
+     * Finds a trip by reasons and date.
+     * @param {string} trip a trip data.
+     * @returns {object} trip data.
+     */
+  static async reasonsDate(trip) {
+    const {
+      userId,
+      reasons,
+      destination,
+      date
+    } = trip;
+    const newDate = new Date(date);
+    const tripExist = await Trip.findOne({
+      where: {
+        userId,
+        reasons,
+        destination,
+        date: newDate
+      }
+    });
+    return tripExist;
+  }
+
+  /**
+     * Saves a trip in the DB.
+     * @param {object} trip The request sent by a user.
+     * @returns {object} trip data.
+     */
+  static async saveTrip(trip) {
+    const acceptedTrip = await Trip.create({
+      ...trip,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    }, {
+      fields: [
+        'userId',
+        'tripType',
+        'departure',
+        'destination',
+        'date',
+        'reasons',
+        'returnDate',
+        'status',
+        'createAt',
+        'updatedAt'
+      ]
+    });
+
+    return acceptedTrip;
+  }
+
+  /* eslint-disable object-curly-newline */
```

Comment:
Fix this ESLint issue instead of disabling it

Reply:
Feedback implemented